package/MDAnalysis/lib/_cutil.pyx

```diff
@@ -211,6 +231,17 @@ def make_whole(atomgroup, reference_atom=None):
             ortho = False
 
     if ortho:
+        # If atomgroup is already unwrapped, bail out
+        is_unwrapped = True
+        for i in range(1, natoms):
+            for j in range(3):
+                if abs(oldpos[i, j] - oldpos[0, j]) >= half_box[j]:
```

Comment:
I'd doublecheck that this `abs` call is producing the right C code, iirc you can sometimes get Python's

Reply:
I guess you could try `from libc.math cimport fabs `, but it may not make much difference.

Reply:
Should be `fabs` anyway